File: update-routes.py
import os
import re

def update_html_files(directory, depth):
    """Actualiza las rutas de assets en los archivos HTML"""
    for filename in os.listdir(directory):
        if filename.endswith('.html'):
            filepath = os.path.join(directory, filename)
            with open(filepath, 'r', encoding='utf-8') as f:
                content = f.read()
            
            # Reemplazar rutas de assets
            new_content = re.sub(
                r'(href|src)="assets/',
                rf'\1="{depth}assets/',
                content
            )
            
            with open(filepath, 'w', encoding='utf-8') as f:
                f.write(new_content)
            
            print(f'✓ Actualizado: {filename}')

# Actualizar las carpetas
base_path = 'c:\\xampp\\htdocs\\Genesis\\paginas'
depth = '../../'

for folder in ['carreras', 'programas-especiales', 'servicios', 'pruebas']:
    folder_path = os.path.join(base_path, folder)
    if os.path.exists(folder_path):
        update_html_files(folder_path, depth)
        print(f'Carpeta {folder} actualizada\n')

print('✓ Todas las rutas han sido actualizadas exitosamente')

File: update-routes.js
const fs = require('fs');
const path = require('path');

function updateHtmlFile(filePath, depth) {
  try {
    let content = fs.readFileSync(filePath, 'utf8');
    // Reemplazar assets/ con ../../assets/
    content = content.replace(/href="assets\//g, 'href="' + depth + 'assets/');
    content = content.replace(/src="assets\//g, 'src="' + depth + 'assets/');
    fs.writeFileSync(filePath, content, 'utf8');
    console.log('✓ ' + path.basename(filePath));
  } catch (err) {
    console.error('✗ Error en ' + filePath + ': ' + err.message);
  }
}

// Procesar todos los HTML en paginas
const dirs = [
  { dir: 'paginas/carreras', depth: '../../' },
  { dir: 'paginas/programas-especiales', depth: '../../' },
  { dir: 'paginas/servicios', depth: '../../' },
  { dir: 'paginas/pruebas', depth: '../../' }
];

dirs.forEach(({dir, depth}) => {
  const folder = path.join(process.cwd(), dir);
  if (fs.existsSync(folder)) {
    fs.readdirSync(folder)
      .filter(f => f.endsWith('.html'))
      .forEach(file => {
        updateHtmlFile(path.join(folder, file), depth);
      });
  }
});

console.log('\nRutas actualizadas exitosamente');
